### DISCUSSION TITLE
Bento Kutusu Macerası

### DISCUSSION THREAD

**Discussion by tolga0102** [sonsuzus/sonsuzus#459] [Soru Cevap] [answered]
## **Bento Kutusu Macerası**

![image](https://github.com/user-attachments/assets/8ba50f90-671d-4305-852e-2b58c5b9d046)

Test başına zaman sınırı: 1 saniye
Test başına bellek sınırı: 1024 MB

Tayvan'da kutulanmış yemekler çok yaygındır, ve öğrenciler ile ofis çalışanları için uygun fiyatlı ve besin açısından dengeli seçenekler sunarlar. Bu yemekler genelde çeşitli sebzeler, protein ve pirinç içerir ve dengeli bir diyet sağlar. Yerel self servis restoranlarında çok sayıda seçenek olduğundan, hızlı ve sağlıklı bir öğle yemeği arayanlar için popüler bir seçimdir.

Problemimizde 1'den 5'e kadar numaralandırılmış beş Tayvanlı self-servis restoranı var. Pazartesiden cumaya her gün farklı bir restoranı ziyaret edip öğle yemeği için bir kutu yemek almayı planlıyorsunuz. Şu ana kadar yani pazartesiden başlayıp perşembeye kadar her seferinde farklı bir restorana gittiniz. Şimdi ise cuma, ve daha önce gitmediğiniz son restorana gitmeyi istiyorsunuz.

Daha önce gittiğiniz dört restoranı (pazartesiden perşembeye kadar her gün için bir tane olmak üzere) girdi olarak alan ve cuma günü gitmeniz gereken restoranı çıktı olarak veren bir program yazın.

**Giriş**
Pazartesiden perşembeye kadar ziyaret ettiğiniz restoran numaralarını sırayla temsil eden, her biri 1 ile 5 (5 dahil) arasında olan a,b,c,d olmak üzere dört tam sayı içeren tek bir giriş satırı.
* 1 <= a,b,c,d <= 5
* Dört sayı da bir birinden farklı olacak.

**Çıktı**
Cuma günü ziyaret etmeniz gereken restoranın numarasını çıktı olarak verin.

**Örnekler**
Giriş:
1 3 2 5
Çıktı:
4
Giriş:
2 5 4 3
Çıktı:
1

**Comment by tolga0102** [comment]
```python a, b, c, d = input().split()
liste = ["1","2","3","4","5"]
liste.remove(a)
liste.remove(b)
liste.remove(c)
liste.remove(d)
e = liste[0]
print(e)
```

**Reply by sonsuzus** [reply]
Çözümde problem var. Test ettin mi? a , b değişkenleri nedir mesela? Girdi olarak verileri alan bir program yapman lazım ayrıca.

**Reply by tolga0102** [reply]
Hm tamam hocam bakarım tekrar

**Comment by tolga0102** [comment, marked as answer]
Hocam pythondan githuba geçirirken ilk satırı kopyalamamışım sanırım, normal halini gönderiyorum :D
```python
a, b, c, d = input().split()
liste = ["1","2","3","4","5"]
liste.remove(a)
liste.remove(b)
liste.remove(c)
liste.remove(d)
e = liste[0]
print(e)
```

**Reply by sonsuzus** [reply]
Aferin ama biraz daha zor soruları tercih etmelisin :)
